lightwood/encoder/numeric/numeric.py

```diff
@@ -1,46 +1,63 @@
 import math
+from typing import Iterable, List, Union
 import torch
 import numpy as np
+from torch.types import Number
 from lightwood.encoder.base import BaseEncoder
 from lightwood.helpers.log import log
 from lightwood.helpers.general import is_none
+from lightwood.api.dtype import dtype
 
 
 class NumericEncoder(BaseEncoder):
+    """
+    The numeric encoder takes numbers (float or integer) and converts it into tensors of the form:
+    ``[is_none(number), is_positive(number), natural_log(abs(number)), number/absolute_mean]``
```

Comment:
you might want to actually mention that is_none and is_positive are in fact boolean outputs of the number itself, maybe even call it inp_number for less ambiguity.

Reply:
done